assignment1/main.py

```diff
@@ -0,0 +1,56 @@
+print("")
+def first_prime_number(number):
+    def is_not_prime(number):
+        flag = False
+        for i in range(2, number):
+            if number % i == 0:
+                flag = True
+        return flag
+
+    while  is_not_prime(number):
+        number = number + 1
+    return number
+
+number = int(input("Enter a number: "))
+print("Problem 1: ")
+print(first_prime_number(number))
+print("")
+
+# Determine the twin prime numbers p1 and p2 immediately larger than the given non-null natural number n.
+# Two prime numbers p and q are called twin if q - p = 2.
+
+def twin_prime(number):
+    first = first_prime_number(number)
```

Comment:
Your functions check for the larger or equal values instead of the strictly larger as asked.